documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,38 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+Setting parameter `invalidateCache` to `true`, will clean up the cache of the deployed application and reflects the new changes.
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
+
+![Oauth client creation](../images/oauthClientCreation.png)
+
+Select the client tab, which provides an option to register a new client as shown below:
+
+![Register new client](../images/registerNewOauthClient.png)
```

Comment:
This information is basically included in the above image, no need to double it IMO.